From leitao@linux.vnet.ibm.com  Thu Oct  8 15:44:18 2009
From: Breno Leitão <leitao@linux.vnet.ibm.com>
Date: Tue, 29 Sep 2009 15:16:57 -0300
Subject: jsm: Remove ch_cpstime field
To: linux-kernel@vger.kernel.org
Cc: Scott.Kilau@digi.com, greg@kroah.com
Message-ID: <70ba6cca73bab63741a4f923f7f0bd38c51b1d5d.1254247209.git.leitao@linux.vnet.ibm.com>


Currently the field jsm_channel->ch_cpstime is defined but never
used, so this patch removes it.

Signed-off-by: Breno Leitão <leitao@linux.vnet.ibm.com>
Cc: Scott Kilau <Scott.Kilau@digi.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/jsm/jsm.h |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/serial/jsm/jsm.h
+++ b/drivers/serial/jsm/jsm.h
@@ -206,8 +206,6 @@ struct jsm_channel {
 
 	u64		ch_close_delay;	/* How long we should drop RTS/DTR for */
 
-	u64		ch_cpstime;	/* Time for CPS calculations	*/
-
 	tcflag_t	ch_c_iflag;	/* channel iflags		*/
 	tcflag_t	ch_c_cflag;	/* channel cflags		*/
 	tcflag_t	ch_c_oflag;	/* channel oflags		*/
